src/mahoji/lib/abstracted_commands/questCommand.ts

```diff
@@ -8,7 +8,7 @@ import addSubTaskToActivityTask from '../../../lib/util/addSubTaskToActivityTask
 import { minionIsBusy } from '../../../lib/util/minionIsBusy';
 import { userHasGracefulEquipped } from '../../mahojiSettings';
 
-export const MAX_GLOBAL_QP = 293;
+export const MAX_GLOBAL_QP = 303;
```

Comment:
This is the max QP excluding the "specific quests", which DT2 is. So you want to use `MAX_QP` (look how its defined to see what i mean)

Reply:
Have updated to add the new quests similar to how dt2 is done